tests/java-db-test/src/main/java/testing/servlets/TestServlet.java

```diff
@@ -0,0 +1,66 @@
+package testing.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This servlet is just to test a query on the Mintern database, and can be used as reference
+ * for later use.
+ */
+@WebServlet("/test")
+public class TestServlet extends HttpServlet {
+  // All the variables needed to connect to the local database.
+  // P.S.: Change the timezone if needed (https://github.com/dbeaver/dbeaver/wiki/JDBC-Time-Zones).
+  String url = "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
+  String user = "root";
+  String password = "";
+
+  // This is the query that will be executed.
+  String query = "SELECT * FROM User";
+  
+  /** 
+   * This method will get the ids and names of all users on the database and return it as
+   * a JSON string.
```

Comment:
Since this is a test servlet and the main purpose is to test SQL queries, I would make the javadoc on this more generalized. This way, other developers can come in and modify it locally to fit their own testing dev needs.

```suggestion
   * This method runs the specified query above on the Mintern database and returns
   * the result as a JSON string.
```

Reply:
done